src/fixup.js

```diff
@@ -139,4 +140,45 @@
     }
   }
 
+  /* Deprecation warning */
+  var request = new XMLHttpRequest();
+
+  request.open('GET', '//www.w3.org/TR/tr-outdated-spec');
```

Comment:
I think you need to be more defensive here... how about:

```JS
request.open("GET", "//www.w3.org/TR/tr-outdated-spec");
request.onload = function() {
  if (request.status <= 200 && request.status > 400) {
    return;
  }
  // Guard against accidentally sending a bad JSON response
  try {
    var currentSpec = JSON.parse(request.responseText);
  } catch (err) {
    console.error(err);
    return;
  }
  document.body.classList.add("outdated-spec");
  var node = document.createElement("p");
  node.classList.add("outdated-warning");

  var warning =
    "<strong>" +
    currentSpec.header +
    "</strong>" +
    currentSpec.warning +
    '<div><a id="outdated-note" href="' +
    currentSpec.latestUrl +
    '"> ' +
    currentSpec.latestUrl +
    '</a>.</div><input onclick="collapseWarning(false)" type="button" value="&#9662; collapse">';
  node.innerHTML = warning;
  document.body.appendChild(node);
};
```

Reply:
`<= 200`? shouldn't that be `<200`?

Reply:
Yeah, sorry 😐